arrow-syntax.api

```diff
@@ -0,0 +1,1632 @@
+public abstract class arrow/core/AndThen0 : kotlin/jvm/functions/Function0 {
+	public static final field Companion Larrow/core/AndThen0$Companion;
+	public final fun andThen (Lkotlin/jvm/functions/Function1;)Larrow/core/AndThen0;
+	public final fun andThenF (Larrow/core/AndThen1;)Larrow/core/AndThen0;
+	public fun invoke ()Ljava/lang/Object;
+	public fun toString ()Ljava/lang/String;
+}
+
+public final class arrow/core/AndThen0$Companion {
+	public final fun invoke (Lkotlin/jvm/functions/Function0;)Larrow/core/AndThen0;
+	public final fun loop (Larrow/core/AndThen0;Ljava/lang/Object;I)Ljava/lang/Object;
+	public final fun rotateAccumulate (Larrow/core/AndThen0;Larrow/core/AndThen1;)Larrow/core/AndThen0;
+}
+
+public final class arrow/core/AndThen0$Single : arrow/core/AndThen0 {
+	public fun <init> (Lkotlin/jvm/functions/Function0;I)V
+	public final fun component1 ()Lkotlin/jvm/functions/Function0;
+	public final fun component2 ()I
+	public final fun copy (Lkotlin/jvm/functions/Function0;I)Larrow/core/AndThen0$Single;
+	public static synthetic fun copy$default (Larrow/core/AndThen0$Single;Lkotlin/jvm/functions/Function0;IILjava/lang/Object;)Larrow/core/AndThen0$Single;
+	public fun equals (Ljava/lang/Object;)Z
+	public final fun getF ()Lkotlin/jvm/functions/Function0;
+	public final fun getIndex ()I
+	public fun hashCode ()I
+	public fun toString ()Ljava/lang/String;
+}
+
+public abstract class arrow/core/AndThen1 : kotlin/jvm/functions/Function1 {
+	public static final field Companion Larrow/core/AndThen1$Companion;
+	public final fun andThen (Lkotlin/jvm/functions/Function1;)Larrow/core/AndThen1;
+	public final fun andThenF (Larrow/core/AndThen1;)Larrow/core/AndThen1;
```

Comment:
We should look into these names with `F` and similar

Reply:
Make these things `private`.